python/plugins/processing/algs/gdal/DatasetIdentify.py

```diff
@@ -0,0 +1,124 @@
+"""
+***************************************************************************
+    DatasetIdentify.py
+    ------------------
+    Date                 : December 2025
+    Copyright            : (C) 2025 by Even Rouault
+    Email                : even dot rouault at spatialys dot com
+***************************************************************************
+*                                                                         *
+*   This program is free software; you can redistribute it and/or modify  *
+*   it under the terms of the GNU General Public License as published by  *
+*   the Free Software Foundation; either version 2 of the License, or     *
+*   (at your option) any later version.                                   *
+*                                                                         *
+***************************************************************************
+"""
+
+__author__ = "Even Rouault"
+__date__ = "December 2025"
+__copyright__ = "(C) 2025, Even Rouault"
+
+import os
+
+from qgis.core import (
+    Qgis,
+    QgsProcessing,
+    QgsProcessingException,
+    QgsProcessingParameterFile,
+    QgsProcessingParameterBoolean,
+    QgsProcessingParameterVectorDestination,
+)
+from processing.algs.gdal.GdalAlgorithm import GdalAlgorithm
+from processing.algs.gdal.GdalUtils import GdalUtils
+
+pluginPath = os.path.split(os.path.split(os.path.dirname(__file__))[0])[0]
+
+
+class DatasetIdentify(GdalAlgorithm):
+    INPUT = "INPUT"
+    RECURSIVE = "RECURSIVE"
+    DETAILS = "DETAILS"
+    OUTPUT = "OUTPUT"
+
+    def __init__(self):
+        super().__init__()
+
+    def initAlgorithm(self, config=None):
+        self.addParameter(
+            QgsProcessingParameterFile(
+                self.INPUT,
+                self.tr("Input folder"),
+                Qgis.ProcessingFileParameterBehavior.Folder,
+            )
+        )
+
+        self.addParameter(
+            QgsProcessingParameterBoolean(
+                self.RECURSIVE,
+                self.tr(
+                    "Perform recursive exploration of the input folder and its subfolders"
```

Comment:
```suggestion
                    "Perform recursive exploration of the input folder"
```
I assume the `recursive` implies that.

Reply:
thanks, fixed